### PR TITLE
[maintainers guide] add link to Kevin's co-author generator

### PR DESCRIPTION
**Pull request by sarah11918**
Adds a link to https://co-author-generator.netlify.app/ in addition to the existing manual instructions for how to add co-author credit when performing the Squash and Merge action.

Built by @kevinzunigacuellar :raised_hands: :rocket:

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
OMG! This is super useful!

Ship it! 🛳️🚀

**Comment by kevinzunigacuellar** [comment]
Credits to @TheOtterlord 💜 (Soon in the footer)

Also I spined up a prod url 🚀 

https://coauthor.kevinzunigacuellar.com/

File: contributor-guides/astro-maintainers-guide-to-docs.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ Typically, this will be an approval from another maintainer, often the Docs Lead
 
 ### `Co-authored-by:` credit
 
+**NEW!** Use [Kevin's Co-author Generator](https://co-author-generator.netlify.app/) to automatically get a list of all contributors to a PR to copy and paste in the "Squash and Merge" message. To add any individual manually, follow the instructions below.
```
Comment (kevinzunigacuellar) [inline contributor-guides/astro-maintainers-guide-to-docs.md:142]:
```suggestion
**NEW!** Use [Kevin's Co-author Generator](https://coauthor.kevinzunigacuellar.com/) to automatically get a list of all contributors to a PR to copy and paste in the "Squash and Merge" message. To add any individual manually, follow the instructions below.
```

**Review by delucis — COMMENTED** [review]
Love it! Thanks for building this @kevinzunigacuellar 🔥

File: contributor-guides/astro-maintainers-guide-to-docs.md (hunk 1 above)
Comment (sarah11918) [reply contributor-guides/astro-maintainers-guide-to-docs.md:142]: Do I need to test this URL to make sure it doesn't, in fact, just make @kevinzunigacuellar a co-author on every PR??  :laughing:

**Review by TheOtterlord — COMMENTED** [review]
Let's get this merged!

**Pull request merged**